pytripgui/canvas_vc/bars/bar.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC
+
+from matplotlib import pyplot as plt
+from matplotlib.axes import Axes
+from matplotlib.pyplot import colorbar
+
+from pytripgui.canvas_vc.bars.projection_enum import BarProjection
+
+
+# base abstract class for bars
+class Bar(ABC, Axes):
+    name: str = BarProjection.DEFAULT.value

```

Comment:
In which python version type annotations were introduced ? We should support python 3.5